pulsar-websocket/src/main/java/org/apache/pulsar/websocket/service/WebSocketProxyConfiguration.java

```diff
@@ -67,9 +67,13 @@
     )
     private String globalZookeeperServers;
 
+    @Deprecated
     @FieldContext(doc = "Connection string of configuration store servers")
     private String configurationStoreServers;
 
+    @FieldContext(doc = "Connection string of configuration metadata store servers")
+    private String configurationMetadataStoreUrl;
```

Comment:
Please help also update the `websocket.conf`, following the same pattern of `broker.conf` to move the deprecated configurations to the end of the config file.